SIO2Arduino.ino

```diff
@@ -63,10 +63,17 @@ LiquidCrystal lcd(PIN_LCD_RD,PIN_LCD_ENABLE,PIN_LCD_DB4,PIN_LCD_DB5,PIN_LCD_DB6,
 #endif
 
 void setup() {
-#ifdef DEBUG
-  // set up logging serial port
-  LOGGING_UART.begin(115200);
-#endif
+  #ifdef DEBUG
+    // set up logging serial port
+    LOGGING_UART.begin(115200);
+    while (!LOGGING_UART) {
+      yield();
+    }
+    LOG_MSG_CR(F("Debug logging enabled"));
+    #ifdef ARDUINO_UNO
+      LOG_MSG_CR(F("WARNING: using debug logging with Uno, which only has 1 UART - SIO will conflict"));

```

Comment:
I threw this in because it took me an embarrassing amount of time to realize the Serial conflict while debugging